projects/valtimo/config/assets/core/de.json

```diff
@@ -1662,6 +1662,10 @@
     "Yes": "Ja",
     "No": "Nein"
   },
+  "booleanTypeConverter": {
```

Comment:
viewTypeConverter already exists for this, either remove this, or remove the other one